### PR TITLE
i18n(fr): Translate `index.mdx`, `frontend/index.mdx`, `nextjs.mdx`, `nuxt.mdx`, `trunk.mdx`

### PR DESCRIPTION
**Pull request by 0ri0nexe**
Here it is, it may needs a double check but
- [fr/index.mdx](https://github.com/0ri0nexe/tauri-docs/blob/next/src/content/docs/fr/index.mdx)
- fr/2/guide/frontend/
   - [index.mdx](https://github.com/0ri0nexe/tauri-docs/blob/next/src/content/docs/fr/2/guide/frontend/index.mdx)
   - [nextjs.mdx](https://github.com/0ri0nexe/tauri-docs/blob/next/src/content/docs/fr/2/guide/frontend/nextjs.mdx)
   - [nuxt.mdx](https://github.com/0ri0nexe/tauri-docs/blob/next/src/content/docs/fr/2/guide/frontend/nuxt.mdx)
   - [trunk.mdx](https://github.com/0ri0nexe/tauri-docs/blob/next/src/content/docs/fr/2/guide/frontend/trunk.mdx)

are translated into french

### CODE REVIEW

File: src/content/docs/fr/2/guide/frontend/index.mdx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+</CardGrid>
+
+## Rust
+
+<CardGrid>
+	<LinkCard title="Leptos" href="fr/2/guide/frontend/leptos" />
+	<LinkCard title="Sycamore" href="fr/2/guide/frontend/sycamore" />
+	<LinkCard title="Trunk" href="fr/2/guide/frontend/trunk" />
+	<LinkCard title="Yew" href="fr/2/guide/frontend/yew" />
+</CardGrid>
```
Comment (lorenzolewis) [inline src/content/docs/fr/2/guide/frontend/index.mdx]:
```suggestion
<CardGrid>
	<LinkCard title="Next.js" href="/fr/2/guide/frontend/nextjs" />
	<LinkCard title="Nuxt" href="/fr/2/guide/frontend/nuxt" />
	<LinkCard title="Qwik" href="/fr/2/guide/frontend/qwik" />
	<LinkCard title="Svelte" href="/fr/2/guide/frontend/svelte" />
	<LinkCard title="Vite" href="/fr/2/guide/frontend/vite" />
	<LinkCard title="Webpack" href="/fr/2/guide/frontend/webpack" />
</CardGrid>

## Rust

<CardGrid>
	<LinkCard title="Leptos" href="/fr/2/guide/frontend/leptos" />
	<LinkCard title="Sycamore" href="/fr/2/guide/frontend/sycamore" />
	<LinkCard title="Trunk" href="/fr/2/guide/frontend/trunk" />
	<LinkCard title="Yew" href="/fr/2/guide/frontend/yew" />
</CardGrid>

```

File: src/content/docs/fr/index.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+hero:
+  image:
+    file: ../../../assets/logo-outline.svg
+  actions:
+    - text: Commencer
+      link: fr/2/guide/
+      icon: right-arrow
+      variant: primary
+    - text: Directives de Contrubution
+      link: fr/contribute/
```
Comment (lorenzolewis) [inline src/content/docs/fr/index.mdx]:
```suggestion
      link: /fr/2/guide/
      icon: right-arrow
      variant: primary
    - text: Directives de Contrubution
      link: /fr/contribute/

```

File: src/content/docs/fr/index.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+<CardGrid stagger>
+	<Card title="Frontend Indépendant" icon="rocket">
+		Importez votre configuration web existant dans Tauri ou démarrez ce nouveau
+		projet de rêve.
+		Tauri supporte n'importe quel framework de frontend, vous n'avez donc
+		pas besoin de changer votre configuration.
+	</Card>
+	<Card title="Multiplatforme" icon="rocket">
+		Développez vos applications pour Linux, macOS, Windows, Android et iOS
+		- tout cela à partir du même code.
```
Comment (lorenzolewis) [inline src/content/docs/fr/index.mdx]:
```suggestion
		Développez vos applications pour Linux, macOS, Windows, Android et iOS - tout cela à partir du même code.

```

Looks like this was being treated as a bullet, so just needs to be put on the same line

**Review by lorenzolewis — CHANGES_REQUESTED** [review]
Today I learned about https://www.babbel.com/en/magazine/french-spacing, in particular with `:` and `?`. Didn't realize that was a thing 🇫🇷 

Just a few changes to links and formatting. I'll leave a translation content review to @lemarier / @david-crabnebula if he's able or another French speaker if they'd like to jump in 💪 

Awesome as always @0ri0nexe !

File: src/content/docs/fr/2/guide/frontend/index.mdx [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+## Instructions de Configuration
+
+Théoriquement Tauri agit comme un hôte web statique. Vous avez besoin de fournir à Tauri un dossier contenant un mix de HTML, CSS, Javascript et possiblement de WASM qui peut servir à la vue web que tauri fourni.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/index.mdx]:
```suggestion
Théoriquement, Tauri agit comme un hôte web statique. Vous avez besoin de fournir à Tauri un dossier contenant un mix de HTML, CSS, Javascript et possiblement de WASM qui peut servir à la vue web que Tauri fourni.

```

File: src/content/docs/fr/2/guide/frontend/index.mdx [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+Ci-dessous se trouve une liste des scénarios communs nécessaires pour intégrer un frontend avec Tauri :
+
+{/* TODO: Link to core concept of SSG/SSR, etc. */}
+{/* TODO: Link to mobile development server guide */}
+{/* TODO: Concept of how to do a client-server relationship? */}
+
+- Utilisez la génération statique de site (SSG). Tauri ne supporte pas officiellent les solutions basées sur serveur (par exemple SSR).
+- Pour le développement mobile, un serveur de développement d'un certain type est nécessaire pour héberger le frontend de votre IP locale.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/index.mdx]:
```suggestion
- Pour le développement mobile, un serveur de développement d'un certain type est nécessaire pour héberger le frontend sur votre IP locale.

```
I also wonder if `d'un certain type` is really useful here

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx [hunk 6]
```diff
@@ -0,0 +1,103 @@
+---
+title: Next.js
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Next.js est un framework de React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nextjs.mdx]:
```suggestion
Next.js est un framework React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.

```

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+title: Next.js
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Next.js est un framework de React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.
+
+## Liste des Vérifications
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nextjs.mdx]:
```suggestion
## Liste de contrôle
```

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+
+- Utilisez des exports statiques en définissant `output: 'export'`. Tauri ne supporte pas les solutions basées sur serveur.
+- Utilisez `internal-ip` pour une compatibilité sur mobile pour pouvoir configurer `assetPrefix`. Ceci est requis pour que le serveur résolve correctement les assets.
+- Utilisez `out/` en tant que `distDir` dans `tauri.conf.json`.
+
+## Configuration d'Exemple
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nextjs.mdx]:
```suggestion
## Exemple de configuration

```

File: src/content/docs/fr/2/guide/frontend/index.mdx [hunk 9]
```diff
@@ -0,0 +1,49 @@
+---
+title: Frontend Configuration
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/index.mdx]:
```suggestion
title: Configuration Frontend

```

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+</Tabs>
+
+3. Mettez à jour la configuration de Next.js :
+
+```ts
+// next.conf.ts
+const isProd = process.env.NODE_ENV === 'production';
+module.exports = async (phase, { defaultConfig }) => {
+	let internalHost = null;
+	// Dans le mode développeur on utilise "internal-ip" pour se servir des assets.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nextjs.mdx]:
```suggestion
	// En mode développement, on utilise "internal-ip" pour se servir des assets.

```

File: src/content/docs/fr/2/guide/frontend/nuxt.mdx [hunk 11]
```diff
@@ -0,0 +1,101 @@
+---
+title: Nuxt
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Apprenez-en plus au sujet de Nuxt sur https://nuxt.com. Ce guide est valable depuis Nuxt 3.7.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nuxt.mdx]:
```suggestion
Apprenez-en plus au sujet de Nuxt sur https://nuxt.com. Ce guide est valable à partir de Nuxt 3.7.

```
I change here to be consistent with the same sentence from NextJS.

File: src/content/docs/fr/2/guide/frontend/nuxt.mdx [hunk 12]
```diff
@@ -0,0 +1,101 @@
+---
+title: Nuxt
+i18nReady: true
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Apprenez-en plus au sujet de Nuxt sur https://nuxt.com. Ce guide est valable depuis Nuxt 3.7.
+
+## Liste des Vérifications
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nuxt.mdx]:
```suggestion
## Liste de contrôle

```

File: src/content/docs/fr/2/guide/frontend/nuxt.mdx [hunk 13]
```diff
@@ -0,0 +1,101 @@
...
+
+## Liste des Vérifications
+
+- Utilisez SSG en définissant `ssr: false`. Tauri ne supporte pas les solutions basées sur serveur.
+- Définisez "host" sur `0.0.0.0`.
+- Utilisez `dist/` en tant que `distDir` dans `tauri.conf.json`.
+- Compilez en utilisant `nuxi generate`.
+- (Optionnel): Désactivez la télémétrie en définisant `telemetry: false` dans `nuxt.config.ts`.
+
+## Configuration d'Exemple
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nuxt.mdx]:
```suggestion
## Exemple de configuration

```

File: src/content/docs/fr/2/guide/frontend/nuxt.mdx [hunk 14]
```diff
@@ -0,0 +1,101 @@
...
+		// Vous pouvez trouver les variables d'environnements additionnelles sur
+		// https://tauri.app/2/reference/environment-variables/
+		envPrefix: ['VITE_', 'TAURI_'],
+		server: {
+			// Tauri requiert un port constant
+			strictPort: true,
+			// Active le serveur de développement pour être visible par les autres appareils pour le développement mobile
+			host: '0.0.0.0',
+			hmr: {
+				// Utiliser le websocket pour le rechargement à chaud
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/nuxt.mdx]:
```suggestion
				// Utilisez le websocket pour le rechargement à chaud

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 15]
```diff
@@ -0,0 +1,48 @@
+---
+title: Trunk
+i18nReady: true
+---
+
+Trunk est un bundler d'applications web WASM pour Rust. Apprenez-en plus au sujet de Trunk sur https://trunkrs.dev. Ce guide est valable à partir de Trunk 0.17.5.
+
+:::caution[Note sur le Support Mobile]
+
+Le développement mobile avec Tauri n'est pas possible actuellement avec la version officielle de Trunk. Si vous développez pour mobile vous allez avoir besoin d'utiliser https://github.com/amrbashir/trunk jusqu'à ce que https://github.com/thedodd/trunk/pull/494 et https://github.com/thedodd/trunk/pull/500 soient fusionnées et disponnible.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
Le développement mobile avec Tauri est actuellement impossible avec la version officielle de Trunk. Si vous développez pour mobile, vous aurez besoin d'utiliser https://github.com/amrbashir/trunk jusqu'à ce que https://github.com/thedodd/trunk/pull/494 et https://github.com/thedodd/trunk/pull/500 soient fusionnées et disponible.

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+i18nReady: true
+---
+
+Trunk est un bundler d'applications web WASM pour Rust. Apprenez-en plus au sujet de Trunk sur https://trunkrs.dev. Ce guide est valable à partir de Trunk 0.17.5.
+
+:::caution[Note sur le Support Mobile]
+
+Le développement mobile avec Tauri n'est pas possible actuellement avec la version officielle de Trunk. Si vous développez pour mobile vous allez avoir besoin d'utiliser https://github.com/amrbashir/trunk jusqu'à ce que https://github.com/thedodd/trunk/pull/494 et https://github.com/thedodd/trunk/pull/500 soient fusionnées et disponnible.
+
+Vous devrez utiliser `cargo install --git https://github.com/amrbashir/trunk` lorsque vous installez Trunk, ensuite définissez `serve.ws_protocol = "ws"` dans `Trunk.toml`.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
Vous devrez utiliser `cargo install --git https://github.com/amrbashir/trunk` pour installer Trunk, ensuite définissez `serve.ws_protocol = "ws"` dans `Trunk.toml`.

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+
+:::
+
+## Liste des Vérifications
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
## Liste de contrôle

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 18]
```diff
@@ -0,0 +1,48 @@
...
+
+- Utilisez SSG, Tauri ne supporte pas officielement les solutions basées sur serveur.
+- Utilisez `address = "0.0.0.0"` pour que le serveur web soit disponible sur le réseau pour le développement mobile.
+- Activez `withGlobalTauri` pour assurer que les APIs de Tauri sont disponibles dans la variable `window.__TAURI__` et peuvent importées en utilisant `wasm-bindgen`.
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
- Activez `withGlobalTauri` pour vous assurer que les APIs de Tauri sont disponibles dans la variable `window.__TAURI__` et peuvent importées en utilisant `wasm-bindgen`.

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 19]
```diff
@@ -0,0 +1,48 @@
...
+
+## Configuration d'Exemple
```
Comment (doctolivier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
## Exemple de configuration

```

File: src/content/docs/fr/index.mdx [hunk 20]
```diff
@@ -0,0 +1,63 @@
+---
+title: Base renforcée pour vos applications web.
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
title: Une fondation robuste pour vos applications web.

```
We can discuss about it, but I think it's more appropriate

File: src/content/docs/fr/index.mdx [hunk 21]
```diff
@@ -0,0 +1,63 @@
...
+template: splash
+hero:
+  image:
+    file: ../../../assets/logo-outline.svg
+  actions:
+    - text: Commencer
+      link: fr/2/guide/
+      icon: right-arrow
+      variant: primary
+    - text: Directives de Contrubution
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
    - text: Règles de contribution

```
Can be also `Guide de contribution`. I think `Directives` is a too strong word here

File: src/content/docs/fr/index.mdx [hunk 22]
```diff
@@ -0,0 +1,63 @@
...
+      icon: open-book
+      variant: minimal
+    - text: Documentation de Tauri 1.0
+      link: https://tauri.app
+      icon: external
+      variant: minimal
+banner:
+  content: |
+    Vous êtes sur l'avant-première du site pour Tauri 2.0 -
+    <a href="https://tauri.app">Aller au site de Tauri 1.0</a>
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
    <a href="https://tauri.app">Aller sur le site de Tauri 1.0</a>

```

File: src/content/docs/fr/index.mdx [hunk 23]
```diff
@@ -0,0 +1,63 @@
...
+
+<div class="hero-bg">
+	<div class="bg-logo"></div>
+	<div class="bg-grad"></div>
+	<div class="bg-grad-red"></div>
+</div>
+
+<CardGrid stagger>
+	<Card title="Frontend Indépendant" icon="rocket">
+		Importez votre configuration web existant dans Tauri ou démarrez ce nouveau
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		Importez votre configuration web existant dans Tauri ou démarrez votre nouveau

```
sound more impactful with this change, like "start __your__ new dream project"

File: src/content/docs/fr/index.mdx [hunk 24]
```diff
@@ -0,0 +1,63 @@
...
+		projet de rêve.
+		Tauri supporte n'importe quel framework de frontend, vous n'avez donc
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		Tauri supporte n'importe quel framework frontend, vous n'avez donc

```

File: src/content/docs/fr/index.mdx [hunk 25]
```diff
@@ -0,0 +1,63 @@
...
+		pas besoin de changer votre configuration.
+	</Card>
+	<Card title="Multiplatforme" icon="rocket">
+		Développez vos applications pour Linux, macOS, Windows, Android et iOS
+		- tout cela à partir du même code.
+	</Card>
+	<Card title="Communication Inter-Processus" icon="rocket">
+		Écrivez votre frontend en Javascript, la logique de l'application en Rust,
+		puis intégrez-là profondément dans le système avec Swift et Kotlin.
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]: I think `intégrez-là` is not really well said, but I don't have any alternative for the moment

File: src/content/docs/fr/index.mdx [hunk 26]
```diff
@@ -0,0 +1,63 @@
...
+	</Card>
+	<Card title="Sécurité Maximale" icon="rocket">
+		Un élément important pour l'équipe Tauri, qui dirige nos priorités et nos plus
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		C'est la première préoccupation de l'équipe Tauri, qui dirige nos priorités et nos plus

```

File: src/content/docs/fr/index.mdx [hunk 27]
```diff
@@ -0,0 +1,63 @@
...
+		grandes innovations.
+	</Card>
+	<Card title="Poids Minimal" icon="rocket">
+		En utilisant le moteur de rendu web natif du système d'exploitation,
+		le poids d'une application Tauri peut être bas comme 600 Ko.
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		le poids d'une application Tauri peut atteindre 600 Ko.

```

File: src/content/docs/fr/index.mdx [hunk 28]
```diff
@@ -0,0 +1,63 @@
...
+	</Card>
+	<Card title="Fonctionne avec Rust" icon="rocket">
+		Avec les performance et la sécurité pour priorités, Rust est le langage
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		Avec la performance et la sécurité au cœur de ses priorités, Rust est le langage

```

File: src/content/docs/fr/index.mdx [hunk 29]
```diff
@@ -0,0 +1,63 @@
...
+		destiné à la nouvelle génération d'applications.
```
Comment (doctolivier) [inline src/content/docs/fr/index.mdx]:
```suggestion
		de la nouvelle génération d'applications.
```

**Review by doctolivier — CHANGES_REQUESTED** [review]
I've made lot of syntactical changes and fixes

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx (hunk 8 above)
Comment (doctolivier) [reply src/content/docs/fr/2/guide/frontend/nextjs.mdx]: We can keep `Configuration d'exemple` but we need to lowercase Exemple here.
It's common in English to capitalize every word in title, not in French.

**Comment by lorenzolewis** [comment]
Amazing! Thank you for the thorough review @doctolivier 🥳

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx (hunk 8 above)
Comment (lorenzolewis) [reply src/content/docs/fr/2/guide/frontend/nextjs.mdx]: Thanks for the feedback. Is this something that we should think to add to a French-specific section of the translation guide for headings?

File: src/content/docs/fr/index.mdx (hunk 3 above)
Comment (0ri0nexe) [reply src/content/docs/fr/index.mdx]: oh yeah didn't thought about it

File: src/content/docs/fr/2/guide/frontend/index.mdx (hunk 4 above)
Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/index.mdx]: Thank you for all of that ^^

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx (hunk 7 above)
Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/nextjs.mdx:12]: I could not find an exact translation, thank you ! However doesn't you think that we need to keep the capital letter at the begining of the words like in all titles ?

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx (hunk 8 above)
Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/nextjs.mdx]: > We can keep `Configuration d'exemple` but we need to lowercase Exemple here. It's common in English to capitalize every word in title, not in French.

Oh okay i didn't thought it was an english only thing

File: src/content/docs/fr/2/guide/frontend/trunk.mdx (hunk 15 above)
Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/trunk.mdx]: I think here we can even put a `s` to `disponible` to be consistent with the plural and with `fusionnées`, fixed others things

File: src/content/docs/fr/2/guide/frontend/index.mdx [hunk 30]
```diff
@@ -0,0 +1,53 @@
+---
+title: Configuration Frontend
+
+i18nReady: true
+---
+
+import { LinkCard, CardGrid } from '@astrojs/starlight/components';
+
+Tauri est "frontend agnostique" et supporte la plupart des frameworks frontend prêts à l'emploi. Cependant, un framework a parfois besoin d'un peu plus de configuration pour l'intégrer à Tauri. Ci-dessous se trouve une liste des frameworks avec les configurations recommandées.
```
Comment (lemarier) [inline src/content/docs/fr/2/guide/frontend/index.mdx:9]:
```suggestion
Tauri est un "frontend agnostique" et supporte la plupart des frameworks frontend prêts à l'emploi. Cependant, un framework a parfois besoin d'un peu plus de configuration pour l'intégrer à Tauri. Ci-dessous se trouve une liste des frameworks avec les configurations recommandées.
```

File: src/content/docs/fr/2/guide/frontend/nextjs.mdx [hunk 31]
```diff
@@ -0,0 +1,106 @@
...
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+import CommandTabs from '@components/CommandTabs.astro';
+
+Next.js est un framework React. Apprenez-en plus au sujet de Next.js sur https://nextjs.org. Ce guide est valable à partir de Next.js 13.4.19.
+
+
+## Liste de contrôle
+
+- Utilisez des exports statiques en définissant `output: 'export'`. Tauri ne supporte pas les solutions basées sur serveur.
```
Comment (lemarier) [inline src/content/docs/fr/2/guide/frontend/nextjs.mdx]:
```suggestion
- Utilisez des exports statiques en définissant `output: 'export'`. Tauri ne supporte pas les solutions basées sur un serveur.

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 32]
```diff
@@ -0,0 +1,52 @@
...
+
+Vous devrez utiliser `cargo install --git https://github.com/amrbashir/trunk` pour installer Trunk, ensuite définissez `serve.ws_protocol = "ws"` dans `Trunk.toml`.
+
+
+:::
+
+## Liste de contrôle
+
+
+- Utilisez SSG, Tauri ne supporte pas officielement les solutions basées sur serveur.
```
Comment (lemarier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
- Utilisez SSG, Tauri ne supporte pas officielement les solutions basées sur un serveur.

```

File: src/content/docs/fr/2/guide/frontend/trunk.mdx [hunk 33]
```diff
@@ -0,0 +1,52 @@
...
+- Utilisez `address = "0.0.0.0"` pour que le serveur web soit disponible sur le réseau pour le développement mobile.
+- Activez `withGlobalTauri` pour vous assurer que les APIs de Tauri sont disponibles dans la variable `window.__TAURI__` et peuvent importées en utilisant `wasm-bindgen`.
```
Comment (lemarier) [inline src/content/docs/fr/2/guide/frontend/trunk.mdx]:
```suggestion
- Activez `withGlobalTauri` pour vous assurer que les APIs de Tauri sont disponibles dans la variable `window.__TAURI__` et peuvent être importées en utilisant `wasm-bindgen`.
```

**Review by lemarier — CHANGES_REQUESTED** [review]

File: src/content/docs/fr/index.mdx (hunk 23 above)
Comment (0ri0nexe) [reply src/content/docs/fr/index.mdx]: yeah for sure, i was following the wording of the english doc ^^

File: src/content/docs/fr/index.mdx (hunk 25 above)
Comment (0ri0nexe) [reply src/content/docs/fr/index.mdx]: I'll try to find something better

Comment (0ri0nexe) [reply src/content/docs/fr/index.mdx]: what about `intégrez le tout` or maybe `intégrez le résultat` ?

Comment (doctolivier) [reply src/content/docs/fr/index.mdx]: I like `intégrez le tout` yeah, good one 👍

File: src/content/docs/fr/2/guide/frontend/trunk.mdx (hunk 15 above)
Comment (doctolivier) [reply src/content/docs/fr/2/guide/frontend/trunk.mdx]: Yes indeed, I miss this one 👍

File: src/content/docs/fr/index.mdx (hunk 23 above)
Comment (doctolivier) [reply src/content/docs/fr/index.mdx]: Ahah no worries ^^ Just a proposition that I think make senses

File: src/content/docs/fr/2/guide/frontend/index.mdx (hunk 30 above)
Comment (doctolivier) [reply src/content/docs/fr/2/guide/frontend/index.mdx:9]: Are you sure about this one? It look weird to me to add `un`

File: src/content/docs/fr/index.mdx (hunk 25 above)
Comment (0ri0nexe) [reply src/content/docs/fr/index.mdx]: I think too, fixed :)

**Comment by 0ri0nexe** [comment]
Big thank you for the help @lorenzolewis & @lemarier, i think now the pull request is done ;)

File: src/content/docs/fr/2/guide/frontend/index.mdx (hunk 30 above)
Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/index.mdx:9]: well i guess `"frontend agnostique"` is a noun phrase but it feels bad to me too, i'm gonna search for the french rule applying here

Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/index.mdx:9]: ok seems like it depends on how we use it, if `frontend agnostique` is an expression, no problem to delete the `un`, however if frontend is a common name and agnostique an adjective, we need `un` and maybe change the structure into `un frontend agnostique`. Tbh I don't really know what it means, so is that a common expression in the programming sphere ?

Comment (lorenzolewis) [reply src/content/docs/fr/2/guide/frontend/index.mdx:9]: It's meaning is "frontend independent" where which frontend you choose doesn't really matter. Does that help clarify?

Comment (0ri0nexe) [reply src/content/docs/fr/2/guide/frontend/index.mdx:9]: > It's meaning is "frontend independent" where which frontend you choose doesn't really matter. Does that help clarify?

Yeah it actually do, it mean a common expresssion, and we can finally delete this `un`, fixed

**Comment by 0ri0nexe** [comment]
btw this last commit delete an english sentence who was still there in from-tauri-1.mdx

**Review by lorenzolewis — APPROVED** [review]
Awesome job as always @0ri0nexe and thanks for hoping in for the review @lemarier and @doctolivier 👓 

Btw @doctolivier I couldn't find you on Discord so if you're there feel free to give us a shout in `#docs` over there if you'd like 👋 Thanks for the help!

**Pull request merged**